### PR TITLE
[https://nvbugs/6166097][fix] Fix CuteDSL NVFP4 EPLB weight layout

### PR DESCRIPTION
**Pull request by nv-xtf**
## Description

Dynamic EPLB can migrate MoE experts by copying shared host weights into device expert slots. For CuteDSL NVFP4 MoE, the device `w3/w1` weights and scales are transformed into the CuteDSL gate/up interleaved layout after loading, but the shared EPLB weights and scales were not prepared with the same layout before registration.

As a result, dynamically migrated expert slots could receive weights in the wrong layout, causing corrupted logits and near-zero DeepSeek-R1 FP4 wideep accuracy.

This PR applies the CuteDSL `w3/w1` interleave to shared EPLB weights and scales before registration, so migrated expert slots match the device layout. It also updates EPLB test coverage and cleans up wideep accuracy/stress configs:
removing the force-accept speculative-decoding override (`TLLM_SPEC_DECODE_FORCE_NUM_ACCEPTED_TOKENS`) from accuracy configs (MTP itself is kept), and disabling accuracy gating in stress-only configs.

## Test Coverage

- Updated MoE EPLB unit test (`test_configurable_moe_multi_gpu_eplb`) to use the CuteDSL NVFP4 quant method for CUTEDSL coverage.
- Increased EPLB update steps in the unit test so online expert migration is actually exercised (a single step passed even with the bug present).
- Verified the affected DeepSeek-R1 FP4 wideep GSM8K accuracy case recovers (~0.01 -> 0.9568):
  - `flexible-extract exact_match: 0.9568`
  - `strict-match exact_match: 0.9568`
- Ran Python compile check for modified Python files.

## PR Checklist

Please review the following before submitting your PR:
- PR description clearly explains what and why. If using CodeRabbit's summary, please make sure it makes sense.
- PR Follows [TRT-LLM CODING GUIDELINES](https://github.com/NVIDIA/TensorRT-LLM/blob/main/CODING_GUIDELINES.md) to the best of your knowledge.
- Test cases are provided for new code paths (see [test instructions](https://github.com/NVIDIA/TensorRT-LLM/tree/main/tests#1-how-does-the-ci-work))
- If PR introduces API changes, an appropriate PR label is added - either `api-compatible` or `api-breaking`. For `api-breaking`, include `BREAKING` in the PR title.
- Any new dependencies have been scanned for license and vulnerabilities
- [CODEOWNERS](https://github.com/NVIDIA/TensorRT-LLM/blob/main/.github/CODEOWNERS) updated if ownership changes
- Documentation updated as needed
- Update [tava architecture diagram](https://github.com/NVIDIA/TensorRT-LLM/blob/main/.github/tava_architecture_diagram.md) if there is a significant design change in PR.
- The reviewers assigned automatically/manually are appropriate for the PR.

- [x] Please check this after reviewing the above items as appropriate for this PR.

## GitHub Bot Help

To see a list of available CI bot commands, please comment `/bot help`.

## Summary by CodeRabbit

* **New Features**
  * Added extensible hook for transforming weight scales in quantized Mixture of Experts models, enabling backend-specific layout optimizations before expert migration.

* **Tests**
  * Enhanced unit test coverage for Mixture of Experts models using CuteDSL and EPLB backends.
  * Updated test parameters and configurations to better validate online expert migration behavior.
  * Increased test iterations to ensure adequate expert migration during online load balancing.

### CODE REVIEW

File: tests/scripts/perf/disaggregated/gb200_wideep_stress-deepseek-r1-fp4_1k1k_ctx2_gen1_dep16_bs128_eplb288_mtp3_ccb-NIXL.yaml [hunk 1]
```diff
@@ -46,7 +46,7 @@ environment:
 profiling:
   nsys_on: false
 accuracy:
-  enable_accuracy_test: true
+  enable_accuracy_test: false
```
Comment (Shixiaowei02) [inline tests/scripts/perf/disaggregated/gb200_wideep_stress-deepseek-r1-fp4_1k1k_ctx2_gen1_dep16_bs128_eplb288_mtp3_ccb-NIXL.yaml:49]: Just wondering why the accuracy test remains disabled after the fix.

Comment (nv-xtf) [reply tests/scripts/perf/disaggregated/gb200_wideep_stress-deepseek-r1-fp4_1k1k_ctx2_gen1_dep16_bs128_eplb288_mtp3_ccb-NIXL.yaml:49]: This stress/perf config keeps `TLLM_SPEC_DECODE_FORCE_NUM_ACCEPTED_TOKENS=3`, which force-accepts unverified speculative tokens. That is useful as a throughput/stress knob, but it is not correctness-preserving, so GSM8K accuracy would be near-zero in this config regardless of this EPLB layout fix.

Accuracy is covered by the dedicated `*_accuracy-*` configs instead. Those configs drop force-accept while keeping online EPLB + CuteDSL enabled, which is the correctness path fixed and validated by this PR.

**Review by xxi-nv — APPROVED** [review]
